src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -3,6 +3,20 @@
 export RUNTIME_PATH=$1
 export EXECUTION_DIR=$(dirname "$0")
 
+function print_info_from_core_file_using_lldb {
+  local core_file_name=$1
+  local executable_name=$2
+  local plugin_path_name="$RUNTIME_PATH/shared/Microsoft.NETCore.App/9.9.9/libsosplugin.so"
+
+  # check for existence of lldb-3.9 on the path
+  hash lldb-3.9 2>/dev/null || { echo >&2 "lldb-3.9 was not found. Unable to print core file."; return; }
+  echo ----- start ===============  lldb Output =====================================================
+  # clrstack can throw exception if there is no managed call stack. so we are doing it at kast
+  echo Printing native call stack followed by managed exceptions and managed call stack.
+  lldb-3.9 -O "settings set target.exec-search-paths $RUNTIME_PATH" -o "plugin load $plugin_path_name" -o "bt all" -o "pe -nested" -o "clrstack -all" -o "quit"  --core $core_file_name $executable_name
```

Comment:
Please try pe/clrstack before the native trace (because the native trace is so large)
If pe/clrstack might fail as you noted here, then make a separate lldb invocation for the native trace.

so it would be
1) pe/clrstakc if any
2) lldb bt
3) gdb

Reply:
Update based on Jan's mail we only need this

lldb
1. pe -nested  
2. clrstack -a -f

then gdb 
3. thread apply all bt full